script.js

```diff
@@ -13,14 +13,35 @@ function closeNav() {
 const submit = document.getElementById('submit');
 submit.addEventListener('click', (e) => {
   const email = document.getElementById('email').value;
+  const name1 = document.getElementById('name').value;
+  const message = document.getElementById('message').value;
   const emailLower = email.toLowerCase();
   const text = document.getElementById('text');
+  const userInputData = {
+    name1,
+    email,
+    message,
+  };
+  const savedData = JSON.stringify(userInputData);
```

Comment:
- [ ] The data should be persistent on the browser even if the user didn't click on the submit button, as for now the data is added to the local storage only when you submit the form, kindly make the changes so the inputs is populated after the page is reloaded